packages/server/src/auth/newuser.ts

```diff
@@ -9,14 +9,17 @@ import { sendOutcome } from '../fhir/outcomes';
 import { systemRepo } from '../fhir/repo';
 import { globalLogger } from '../logger';
 import { getUserByEmailInProject, getUserByEmailWithoutProject, tryLogin } from '../oauth/utils';
-import { bcryptHashPassword, getProjectByRecaptchaSiteKey, verifyRecaptcha } from './utils';
 import { makeValidationMiddleware } from '../util/validator';
+import { bcryptHashPassword, getProjectByRecaptchaSiteKey, verifyRecaptcha } from './utils';
 
 export const newUserValidator = makeValidationMiddleware([
-  body('firstName').notEmpty().withMessage('First name is required'),
-  body('lastName').notEmpty().withMessage('Last name is required'),
-  body('email').isEmail().withMessage('Valid email address is required'),
-  body('password').isLength({ min: 8 }).withMessage('Password must be at least 8 characters'),
+  body('firstName').isLength({ min: 1, max: 72 }).withMessage('First name must be between 8 and 72 characters'),
+  body('lastName').isLength({ min: 1, max: 72 }).withMessage('Last name must be between 8 and 72 characters'),
```

Comment:
The error messages are out of sync with the actual limits here, but also these limits are probably not needed, since these inputs are not hashed

Reply:
Good catch on the numeric mismatch, will fix.

On whether the limits are actually required.... We received a bug report for DDoS risk due to open storage of these fields, on unauthenticated endpoints.  The reporter sent a video which uploaded a 1 MB text file in the name field.  It's a reasonable callout.